posthog/migrations/0192_event_properties.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 3.2.5 on 2021-12-03 09:12
+
+import django.contrib.postgres.indexes
+import django.db.models.deletion
+import django.utils.timezone
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0191_rename_specialmigration_asyncmigration"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="EventProperty",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("event", models.CharField(max_length=400)),
+                ("property", models.CharField(max_length=400)),
+                (
+                    "property_type",
+                    models.CharField(
+                        choices=[
+                            ("NUMBER", "Number"),
+                            ("STRING", "String"),
+                            ("BOOLEAN", "Boolean"),
+                            ("DATETIME", "DateTime"),
+                        ],
+                        default="STRING",
+                        max_length=20,
+                        null=True,
+                    ),
+                ),
+                ("property_type_format", models.CharField(max_length=100, null=True)),
+                ("total_volume", models.BigIntegerField(default=None, null=True)),
```

Comment:
Is a `bigint` (goes up to `9223372036854775807`) enough here, or should we use the slower `decimal` that goes up to `131072` digits?

Reply:
Assuming that's not tongue in cheek. That's 9 quintillion. The internet tells me 1 quintillion seconds is 32 billion years.

I guess we'd run out of disk space before we stored that many events 🤣